internal/io/sql/column.go

```diff
@@ -0,0 +1,133 @@
+package sql
+
+import (
+	"fmt"
+	"math"
+	"reflect"
+
+	"github.com/tobgu/qframe/errors"
+)
+
+// Column implements the sql.Scanner interface
+// and allows arbitrary data types to be loaded from
+// any database/sql/driver into a QFrame.
+type Column struct {
+	kind  reflect.Kind
+	nulls int
+	// pointer to the data slice which
+	// contains the inferred data type
+	ptr  interface{}
+	data struct {
+		Ints    []int
+		Floats  []float64
+		Bools   []bool
+		Strings []string
+	}
+}
+
+// Null appends a new Null value to
+// the underlying column data.
+func (c *Column) Null() error {
+	// If we haven't inferred the type of
+	// data we are scanning simply count
+	// the number of NULL values we receive.
+	// The only scenario this will happen is
+	// when the first returned values are NULL.
+	if c.kind == reflect.Invalid {
+		c.nulls++
+		return nil
+	}
+	switch c.kind {
+	case reflect.Float64:
+		c.data.Floats = append(c.data.Floats, math.NaN())
+	case reflect.String:
+		c.data.Strings = append(c.data.Strings, "")
```

Comment:
Is this the way you want null-strings to be handled (eg. as empty strings)? Instead of passing `[]string` to `qframe.New()` you could pass `[]*string` to allow nil-values. I'll make sure to add an example of this to the documentation to make it more obvious.